website/source/docs/platform/k8s/helm.html.md

```diff
@@ -235,6 +257,15 @@ and consider if they're appropriate for your deployment.
   [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#resourcerequirements-v1-core) object. If this isn't specified, then the pods
   won't request any specific amount of resources.
 
+  ```yaml
+  # Resources are defined as a formatted multi-line string:
+  resources: |
+    requests:
+      memory: "10Gi"
```

Comment:
Does it make sense that both client and servers have the same memory example? Could be a chance to demonstrate something alternate.

Reply:
I initially wasn't going to add an example for both, but then realized people might only use one or the other. I'm okay using the same thing for now, since we haven't figured out recommended resource requirements, but I'll definitely keep this in mind when I figure out what are recommendations are :).